# Chapter 9 : Excption

print('Start')

x = 100
try:
    print(y)
except:
    print('Something went wrong.')
else:
    print('Nothing went wrong.')
finally:
    print('Hello from finally block.')

print('End')